gradle/java.gradle

```diff
@@ -73,9 +73,9 @@ gradle.taskGraph.whenReady({ graph ->
       }
     }
     jar.metaInf {
-      from("$rootDir/LICENSE")
+      from("$rootDir/geode-assembly/src/main/dist/LICENSE")
       if (jar.source.filter({ it.name.contains('NOTICE') }).empty) {
-        from("$rootDir/NOTICE")
+        from("$rootDir/geode-assembly/src/main/dist/NOTICE")
```

Comment:
I'm not 100% sure on this one.  If there are no bundled libs (e.g. for geode jars) using the source LICENSE/NOTICE files seems better.  For war files using the full contents of LICENSE/NOTICE from geode-assembly seems better.  But I could be convinced otherwise.

Reply:
I think gradle treats .war and .jar files as equivalent for applying this clause.  Maybe @rhoughton-pivot can offer some more insight.  I confess my understand of what NOTICE goes where is not as clear as my understanding of LICENSE...

Reply:
I have reverted this change.  Sounds like NOTICE cleanup might be best for a separate PR.

Reply:
Makes sense